packages/next-auth/src/lib/env.ts

```diff
@@ -0,0 +1,48 @@
+import { NextRequest } from "next/server.js"
+import type { headers } from "next/headers.js"
```

Comment:
> Do you mean find/replace bugs on my code? or in this branch?

@kzlar I assume something like this would help in this branch:

```suggestion
import { NextRequest } from "next/server"
import type { headers } from "next/headers"
```

So either fork or create a patch for this or wait for the devs to fix this. It seems like one of the lines I quoted earlier have already been fixed.

Reply:
thank you. I manually changed the js files and now I'm having the same issue as kevva and asterikx above.